src/stores/activityStore.ts

```diff
@@ -15,19 +12,11 @@ export class ActivityStore {
   @observable commonCards = ["", "", "", "", ""];
   @observable playersCards: string[][] = [["", ""]];
 
-  @action setCommonCards(newCards: string[]) {
-    this.commonCards = newCards;
+  @action setCommonCards(newCards: CardArrayInterface[]) {
+    this.commonCards = changeCardNaming([newCards])[0] as string[];
   }
-  @action setPlayerCards(newPlayersCards: any[]) {
+  @action setPlayerCards(newPlayersCards: CardArrayInterface[][]) {
```

Comment:
I would ignore `Array` and `Interface` in the name. If this is an array you can use `[]` which indicates that. Since this is interface you don't need to explicitly name it that way. You can simply go with `Card[]` or `PlayerCard[]`